src/utilities/conditionals.ts

```diff
@@ -24,4 +24,12 @@ export function settingsChanged(settingsPayload: ComputationMenuPayload, compari
        settingsPayload.districtSeats !== comparison.districtSeats ||
        comparison.areaFactor !== settingsPayload.areaFactor
    );
+}
+
+export function assert(condition: boolean, message?: string) {
+    const errorMessage = message || "Assertion failed";
+    if (!condition) {
+        throw new Error(errorMessage);
+        
+    }
```

Comment:
Where is this used?

Reply:
I was using it in the implementation at one point, but removed the code some time later. Must have forgotten to remove this implementation.